### ISSUE TITLE
Replace character key shortcuts

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/calendar#4366]
### Is your feature request related to a problem? Please describe.

Accessibility issue.

There are keyboard shortcuts for activating different functions in the Calendar app. Shortcuts such as these can interfere with shortcuts used by some assistive technologies. 

![image](https://user-images.githubusercontent.com/6078378/178685926-d40c2063-f627-4c27-ad0e-fb250932ace7.png)

Because of this the character key shortcuts should be replaced.

Suggested solution
- Give an option to turn off keyboard shortcuts
- Set them to only be active when the user is in the area they affect
- Test so the shortcuts don’t interfere with screen reader commands

Please make sure that the shortcuts don’t interfere with screen reader commands (NVDA, Jaws, Orca).

See more here 
https://www.w3.org/TR/WCAG21/#character-key-shortcuts
https://webtest.bitv-test.de/index.php?a=di&iid=269&s=n

**Comment by nickvergessen** [comment]
Server PR for a global setting to opt out of keyboard shortcuts to satisfy accessibility is in:
https://github.com/nextcloud/server/pull/34081

Initial state to check is: `loadState('theming', 'shortcutsDisabled', false)`

If that is `true` the app(s) must not register ANY shortcuts.

**Comment by szaimen** [comment]
> If that is `true` the app(s) must not register ANY shortcuts.

I disagree. Shortcuts with modifier key are always okay IIRC.

cc @jancborchardt

**Comment by nickvergessen** [comment]
> I disagree. Shortcuts with modifier key and probably also ESC are always okay IIRC.

This was the outcome of the accessibility review. We have a global setting which disables everything and done.
It heavily depends on the screenreader and tools that you use if Ctrl and/or Alt or other things are okay to use or not. And since maintaining a more detailed list is too much effort, we have a global on/off
